src/components/ListWraper/ListItem/ListItem.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+//import PropTypes from 'prop-types';
+
+const ListItem = ({id, name, surname, location, age}) => (
+  <tr dataRowToRemove={id}>
```

Comment:
co robi ten prop dataRowToRemove??

Reply:
miało za zadanie usuwać wiersz, teraz widzę, że tutaj działa to zupełnie inaczej, zatem zrobię komponent, który będzie odpowiedzialny za usunięcie / zmianę listy w state